src/main/features/dashboard/views/index.njk

```diff
@@ -23,39 +23,39 @@
           <th scope="col">{{ t('Defendant name') }}</th>
           <th scope="col">{{ t('Claim amount') }}</th>
 
-          <th scope="col" class="mobile-hide">{{ t('Status') }}</th>
+          <th scope="col" class="phone-hide">{{ t('Status') }}</th>
         </tr>
         </thead>
         <tbody>
         {% if claimDraftSaved %}
-          <tr class="draft mobile-table">
+          <tr class="draft phone-table">
```

Comment:
this should remain as mobile-table as it relates to how this is displayed when a user in on a mobile and the other places in this file